src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
```

Comment:
[CODE STYLE] - As with the `add` method, use the assignment operator to subtract the value. Change `return a - b;` to `a -= b; return a;`.